molecule/resources/converge.yml

```diff
@@ -6,6 +6,7 @@
 
   vars:
     openwisp2_network_topology: true
+    openwisp2_controller_pip: "https://github.com/pandafy/openwisp-controller/tarball/fix-tests"
```

Comment:
`pandafy`?

Reply:
Once https://github.com/openwisp/openwisp-controller/pull/330 is merged, we can just remove the topmost commit and merge. The work here is complete and ready to be tested. :smile: